src/json.human.js

```diff
@@ -132,8 +144,20 @@
             for (key in data) {
                 isEmpty = false;
 
+                valNode = _format(data[key], options, key);
                 keyNode = sn("th", OBJ_KEY_CLASS_NAME, key);
-                valNode = scn("td", OBJ_VAL_CLASS_NAME, _format(data[key]));
+
+                // Is Hyperlink Key
+                if( options.hyperlinks.enable &&
```

Comment:
can you hold a reference to hyperlinks and keys in a variable to avoid repeated options object lookups? (I use it in cases where the rendered object is huge :)